tests/pytorch/test_onnx_export.py

```diff
@@ -1134,6 +1134,7 @@ def test_export_layernorm_mlp(
 
 
 @skip_FP8
+@pytest.mark.skipif(True, reason="ONNX not accepting string argument qkv_format")
```

Comment:
Are we breaking ONNX support?

Reply:
I made changes similar to those in PR 437. Should be passing now!